### DISCUSSION TITLE
Translating the remix docs

### DISCUSSION THREAD

**Discussion by machour** [remix-run/remix#3686] [General]
As of today, we're still waiting for the [remix router to be remixed](https://remix.run/blog/remixing-react-router) to happen, and the documentation to be updated before kicking off the translation work.

In the mean time, I'm opening this conversation so that volunteers can make themselves known, and be notified when it's time.

Please state in comments the language you'll be able to help with (preferably your first language or native language).

**Comment by machour** [comment]
I'll be available for the French (fr) translation.

**Reply by hakimLyon** [reply]
Me too

**Comment by CodePsy-2001** [comment]
I'll be available for the Korean (kr) translation.

**Comment by lili21** [comment]
I'll be available for the Chinese(zh-CN) translation.

**Reply by Byacrya** [reply]
Same!

**Reply by tiancihe** [reply]
Same here

**Reply by KyrieLii** [reply]
me too

**Comment by sergiodxa** [comment]
I’ll be available for the Spanish (es, es-AR) translation

**Comment by ronnylt** [comment]
II’ll be available for the Spanish (es, es-ES) translation.

**Comment by lgastler** [comment]
II’ll be available for the German (de, de-DE) translation.

**Reply by mrksbnch** [reply]
I can also help out with the German translation.

**Comment by felipefreitag** [comment]
I'm available for the Portuguese (pt-br) translation.

**Reply by Mario-aj** [reply]
Me too.

**Reply by gustavoguichard** [reply]
Me too

**Reply by exploitmik** [reply]
me too

**Reply by rbgfloripa** [reply]
me too

**Reply by itodevio** [reply]
Me too!

**Comment by Byacrya** [comment]
I can help with Simplified Chinese (zh-cn).

**Comment by kayac-chang** [comment]
I'll be available for the Traditional Chinese(zh-Hant) translation.

**Reply by MnHung** [reply]
That interests me as well.

**Comment by kentcdodds** [comment]
Thank you all so much for your willingness! We're working on a rework/reorg of the website right now. Part of what we're doing will make it easier to translate. We don't want to start with a sub-par experience so we appreciate your patience here!

**Comment by andersonFaro9** [comment]
I'll be available for the Brazil (BR) translation.

**Comment by nullndr** [comment]
I am available for Italian (IT) translations

**Comment by supachaidev** [comment]
I’ll be available for the Thai (th) translation.

**Comment by CodePsy-2001** [comment]
I think it's been quite a while since this topic was first proposed. When can I start working on the translation? In my opinion, regardless of the specific function of the framework, translation work should be preceded.

**Reply by machour** [reply]
We're still figuring out how to best intertwine Remix & React Router docs, before starting the translation effort. 

I'm working on a POC that I'll be submitting to the team this week, hopefully things will get moving once we reach consensus.

**Comment by hrutik7** [comment]
I’ll be available for the Hindi (hi) translation.

**Comment by arisa-fukuzaki** [comment]
I'll be available for Japanese (ja) translation🙌

**Reply by takagimeow** [reply]
Me too🙌

**Comment by fdemb** [comment]
I'll be available for Polish (pl-PL) translations!

**Comment by utkarshpant** [comment]
I'm available for Hindi (hi) translations as well!
